src/encoded/static/components/cart/search_listing.js

```diff
@@ -0,0 +1,141 @@
+// node_modules
+import React from 'react';
+import PropTypes from 'prop-types';
+import { motion } from 'framer-motion';
+import marked from 'marked';
+import { sanitize } from 'dompurify';
+// libs
+import { svgIcon } from '../../libs/svg-icons';
+// components
+import { auditDecor } from '../audit';
+import { PickerActions, resultItemClass } from '../search';
+import Status from '../status';
+
+
+/** Height of collapsed description in pixels */
+const DESCRIPTION_COLLAPSED_HEIGHT = 120;
+
+
+/**
+ * Defines the animations for expanding/collapsing a description.
+ */
+const descriptionVariants = {
+    expanded: { height: 'auto' },
+    closed: { height: `${DESCRIPTION_COLLAPSED_HEIGHT}px` },
+};
+
+
+/**
+ * Display the description of a cart in the cart search results. Include a control to collapse and
+ * expand the description when the description has more text than would fit in the allowed vertial
```

Comment:
"vertical". It would be cool if we had the power like Shakespeare to invent new words rather than limiting ourselves to a dictionary.